content/stranger_case/report.md

```diff
@@ -0,0 +1,175 @@
+
+![Tosalios](/images/stranger_case/coc.png)
+
+Once on the game, we can find him using his player id #88PPOOGG2. If we look at his clan details we find a link to a Discord server: [https://discord.gg/65CuhRczxj](https://discord.gg/65CuhRczxj)
+We need to prove we have joined the clan if we want to get some access role on the server, but we did not find anything interesting. However, by chatting with Raphael,
+and by pretending having the same passion for cycling, we managed to get a special role on the server, and we got access to specials channels.
+In the secret channel we found out he is second in
+command at the company. Also, the company received threatening emails
+for some time, and it kept getting worse. He forwarded email from Hortense:
+![Private message](/images/stranger_case/private.png)
+![Email](/images/stranger_case/threat.png)
+
+
+Thanks to this email we can now add extra information for the profile of [Hortense](#hl).
+Also, the sender of the mail use the following email: unknownuser89938487@proton.me
+
+
+
+#### Ulrich Jablonowski {#uj}
+
+{{< imgWithClass "/images/stranger_case/uj.png" "Ulrich Jablonowski" "uj.png" "" "stranger-case-pp" >}}
+
+- **Linkedin**: [https://www.linkedin.com/in/ulrich-jablonowski-0a17a5278/](https://www.linkedin.com/in/ulrich-jablonowski-0a17a5278/)
+
+Suspicious because he says the past few weeks have been good while all other received threats.
+
+#### Faye Terni {#ft}
+
+{{< imgWithClass "/images/stranger_case/ft.png" "Faye Terni" "ft.png" "" "stranger-case-pp" >}}
+
+- **Facebook**: [https://www.facebook.com/profile.php?id=100088501506551](https://www.facebook.com/profile.php?id=100088501506551)
+- **Linkedin**: [https://www.linkedin.com/in/faye-terni-9a7508259/](https://www.linkedin.com/in/faye-terni-9a7508259/)
+- **Email**: faye.terni@gmail.com
+
+Her older Facebook profile picture has a special instruction with hexiftool:
+FBMD0a000a700100004d1400002236000078390000853d00006449000085690000e96c0000cf7100002e7700000eae0000
+
+#### Hortense Lutero {#hl}
+
+{{< imgWithClass "/images/stranger_case/hl.png" "Hortense Lutero" "hl.png" "" "stranger-case-pp" >}}
+
+- **Linkedin**: [https://www.linkedin.com/in/hortense-lutero-ab3507259/](https://www.linkedin.com/in/hortense-lutero-ab3507259/)
+- **Professional email**: hortense.lutero@copprethia.fr
+- **Email**: lutero.hortense@gmail.com
+
+
+Was threatened in an email sent by
+anonymous user, they want to buy a
+property next to where she lives. Due to email, she said she changed
+phone numbers, we have personal
+email address of her, and will leave
+town. She lives at 87 Bd Laennec, Lorient, France
```

Comment:
Put all this in a single line. Grammar could be better but it's fine. Also she lives at both 87 Bd and 85. She owns both.

Reply:
okey done